Yoast/wordpress-seo
# Add tracking opt-in to Yoast free, opt-out to premium

## Context


* Add tracking to our config wizard so we can enable tracking on Yoast SEO free installs.

## Summary

![Screenshot 2020-06-26 at 11 52 12](https://user-images.githubusercontent.com/487629/85846886-d1c4a100-b7a6-11ea-9d6b-8b1622374f13.png)


This PR can be summarized in the following changelog entry:

* We've added the option for users to opt-in to allow Yoast to track some data about their site.
* [wordpress-seo-premium] We've added the option to opt-out of our usage tracking.

## Relevant technical choices:

* In the process I've removed two unused config wizard fields and [fixed a bug in the monorepo](https://github.com/Yoast/javascript/pull/748).

## Test instructions

This PR can be tested by following these steps:

Two spots:
* See step 7 of the config wizard. Note that this step will default to "Yes" when Premium or any add-on is installed, and "No" otherwise.
* See the new feature toggle under SEO -> Features.

## UI changes

## Documentation

## Quality assurance


Fixes UX-98
